Authors:  Cecil Curry <leycec@gmail.com>
Upstream: no
Synopsis: Hamcrest's "build.xml" file explicitly adds source files to the jar
files it produces. This is quite silly, and we patch it from doing so.
--- old/build.xml	2008-11-15 23:15:07.000000000 +0000
+++ new/build.xml	2010-04-05 06:00:49.707911081 +0000
@@ -236,10 +236,6 @@
                 </classpath>
             </javac>
             <copy file="LICENSE.txt" todir="build/temp/@{destjar}.contents"/>
-            <!-- Put Java source in Jars for user's convenience. -->
-            <copy todir="build/temp/@{destjar}.contents">
-                <fileset dir="@{srcdir}"/>
-            </copy>
             <jar jarfile="@{destjar}">
                 <fileset dir="build/temp/@{destjar}.contents"/>
             </jar>
